zerver/lib/push_notifications.py

```diff
@@ -46,11 +46,11 @@
 DeviceToken = Union[PushDeviceToken, RemotePushDeviceToken]
 
 # We store the token as b64, but apns-client wants hex strings
-def b64_to_hex(data: bytes) -> str:
+def b64_to_hex(data: str) -> str:
     return binascii.hexlify(base64.b64decode(data)).decode('utf-8')
 
-def hex_to_b64(data: str) -> bytes:
-    return base64.b64encode(binascii.unhexlify(data.encode('utf-8')))
+def hex_to_b64(data: str) -> str:
+    return base64.b64encode(binascii.unhexlify(data)).decode()
```

Comment:
Returning `str` instead of `bytes` here is a runtime behavior change to note. I think that’s what it should be, to be consistent with `AbstractPushDeviceToken.token` which is declared as a `CharField`. But I’m not sure how to test.

Reply:
I think this is tested as is, because this function is actually only used in test code (and its reverse only has type changes, not behavior changes).